src/AbstractContainer.php

```diff
@@ -33,20 +33,23 @@
  * Laminas\Navigation\Container
  *
  * AbstractContainer class for Laminas\Navigation\Page classes.
+ *
+ * @template TPage of AbstractPage
+ * @template-implements RecursiveIterator<string, TPage>
  */
 abstract class AbstractContainer implements Countable, RecursiveIterator
 {
     /**
      * Contains sub pages
      *
-     * @var array
+     * @var array<string, AbstractPage>
```

Comment:
Not using `TPage` here?

Reply:
b8361eacac8bbafec0afc9b6d0b23c8ef97bfe21